.github/workflows/push_images.yml

```diff
@@ -0,0 +1,81 @@
+name: Push Latest Images
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/push_images.yml
+      - examples/**
+  workflow_dispatch:
```

Comment:
What happens if you manually trigger this workflow? Can you specify which images to build and push?

Reply:
Good point. I could add inputs for the workflow dispatch and add it to the conditionals so that it could be specified on the manual trigger.